dist/docs/guide/customization/promote-customization.adoc

```diff
@@ -0,0 +1,44 @@
+### Promoting custom artifacts
+
+The fist step to apply changes from a bundle to a server, is to promote the customized artifacts into the custom channel.
+
+Promoting is a two-step process:
+
+ . all artifacts from a bundle are deployed in the custom Maven repository.
+ . update the channel file with new versions of customized artifacts and deploy it in the repository with incremented version.
+
+#### Register custom channel
+*Single server channel*
+
+```
+prospero channel initialize --dir=<PATH/TO/SERVER>
+```
+A server can use a filesystem repository and auto-generated channel to provide customizations. This command will create a Maven repository in `<PATH/TO/SERVER>/.installation/` folder and generate empty customization channel. Both the repository and channel will be then added to the server's configuration. This channel should only be used with a single server.
```

Comment:
Maybe I would put the explanation first and the command later, but I haven't seen how it looks in HTML yet.